src/utils/getElementId.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * SPDX-FileCopyrightText: 2018 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+// make sure the id is unique
+window._nc_vue_element_id = window._nc_vue_element_id ?? 0
+
+/**
+ * Get the next element id for use with HTML / CSS.
+ * We can not be sure that the app has a prefix setup so we prefix ourself
+ */
+export function getElementId() {
+	return `nc-vue-${window._nc_vue_element_id!++}`
+}
```

Comment:
Maybe it's just me, but `get*` feels like it only gets some defined value, not generates/calcualtes with a side effect.

What about `generateElementId`/`createElementId` (though, a little bit long)?

Reply:
TBH: I do not care :sweat_smile: 
So will go with your suggestion.